keys/README.md

```diff
@@ -0,0 +1 @@
+This folder contains the identity files.
```

Comment:
Honestly, my preference is to just use the default of `~/.ssh/` and remove this option, but what you proposed works too.